blockprotocol/blockprotocol
# Consolidate API key pages, update routes and add missing font-family

## 🌟 What is the purpose of this PR?

This PR 
- Removes duplicated code (for API page) from database
- Fixes wrong font-family usage on settings
- Updates `/settings/*` routes to `/account/*`

## 🔗 Related links


- [Asana task](https://app.asana.com/0/1204086167143889/1204260759309476/f) _(internal)_



## ❓ How to test this?

1.  Checkout the branch / view the deployment
1.  Check "My Account" pages and confirm they look like in the design (not the page content, the menu titles), and routes are updated from `/settings/*` to `/account/*`
